src/Panel.vue

```diff
@@ -290,12 +290,6 @@
       this.localMinimized = this.minimizedBool;
     },
     mounted() {
-      this.$nextTick(function () {
-        if (this.hasSrc && (this.preloadBool || this.expandedBool)) {
-          this.$refs.retriever.fetch()
-        }
-      });
-
```

Comment:
Let's update comment:

```diff
- // Set local data to computed prop value
+ // Set local data to computed prop value, triggering retrieveOnOpen in watch callback

Reply:
I've refactored the code such that watch is not needed anymore.